benches/benchsuite/src/lib.rs

```diff
@@ -0,0 +1,194 @@
+use cargo::Config;
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use url::Url;
+
+// This is an arbitrary commit that existed when I started. This helps
+// ensure consistent results. It can be updated if needed, but that can
+// make it harder to compare results with older versions of cargo.
+const CRATES_IO_COMMIT: &str = "85f7bfd61ea4fee08ec68c468762e886b2aebec6";
+
+pub struct Bench {
```

Comment:
The name feels a bit off as this isn't the bench itself but something to setup bench fixtures.

Maybe `Fixtures`?
(if a macro is made, we'd want the macro's name to be updated as well)